realm/realm-library/src/objectServer/java/io/realm/SyncManager.java

```diff
@@ -303,4 +336,5 @@ static void simulateClientReset(SyncSession session) {
     protected static native void nativeInitializeSyncManager(String syncBaseDir);
     private static native void nativeReset();
     private static native void nativeSimulateSyncError(String realmPath, int errorCode, String errorMessage, boolean isFatal);
+    private static native void nativeNotifyNetworkIsBack();
```

Comment:
why name it as `NotifyNetworkIsBack()` instead of `reconnect()` to match the same thing in the Object Store? Are we going to do more things than reconnect in this function?